tests/libgit2/status/worktree.c

```diff
@@ -1360,3 +1360,12 @@ void test_status_worktree__at_head_parent(void)
 	git_tree_free(parent_tree);
 	git_status_list_free(statuslist);
 }
+
+void test_status_worktree__skip_hash(void)
+{
+	git_repository *repo = cl_git_sandbox_init("status_skipHash");
```

Comment:
I'd like this to not be bicapitalized; it doesn't fit with our other unit tests.

```suggestion
	git_repository *repo = cl_git_sandbox_init("status_skiphash");
```